dataprofiler/tests/profilers/profiler_options/test_profiler_options.py

```diff
@@ -103,6 +101,8 @@ def test_numerical_stats_option(self, *mocks):
                 self.assertNotEqual(0, profile_column["statistics"]["variance"])
                 self.assertIsNotNone(
                     profile_column["statistics"]["quantiles"][0])
+                self.assertTrue(profile_column["statistics"]["skewness"] is np.nan)
+                self.assertTrue(profile_column["statistics"]["kurtosis"] is np.nan)
```

Comment:
instead of assert is nan, can we use `assertEqual`?

Reply:
would `assertIsNone` catch this or no since its `np.nan`?

Reply:
I've tried both assertEqual and assertIsNone, but these both don't behave as expected for np.nan